baybe/recommenders/pure/nonpredictive/sampling.py

```diff
@@ -52,13 +54,19 @@ def __str__(self) -> str:
         return to_string(self.__class__.__name__, *fields)
 
 
+@define(kw_only=True)
 class FPSRecommender(NonPredictiveRecommender):
     """An initial recommender that selects candidates via Farthest Point Sampling."""
 
     # Class variables
     compatibility: ClassVar[SearchSpaceType] = SearchSpaceType.DISCRETE
     # See base class.
 
+    initialization: Literal["farthest", "random"] = field(
```

Comment:
For classes, we do things a little differently because there is an elegant way to using converters. Instead of typing things here as literal, please:
* Define enum with the two modes with members and values both in capital letters
* Add the enum here as a type hint
* Add the enum also as a converter and remove the validator

That way, people can both specify enum members OR the strings, but internally we always have the structured enum version

Reply:
thx for changing